New package: ETOPO v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: ETOPO
- Repository: https://github.com/GeoRegionsEcosystem/ETOPO.jl
- Created by: @natgeo-wong
- Version: v0.1.0
- Commit: f5db8b5257edcf754334bd7360dbf709df1ceb01
- Reviewed by: @natgeo-wong
- Reference: https://github.com/GeoRegionsEcosystem/ETOPO.jl/commit/f5db8b5257edcf754334bd7360dbf709df1ceb01#commitcomment-153012807
- Description: Downloading ETOPO datasets
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
* A package for the download, extraction and manipulation of ETOPO relief model data
* For small regions, extract ETOPO data only for your [GeoRegion](https://github.com/GeoRegionsEcosystem/GeoRegions.jl) of interest, without needing to download the whole global dataset!
* Soon-to-come: Filtering a LandSea mask based on a Land-Use dataset, not ETOPO topography
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6797d8ffe4ed1eb8bd8dacba2ba448e416d9a7e80c46dd75eb873f3f44cf0f92b4b1dd9c21998822b5074bd1bf47aaea97a06ccc054bec0ecb688e88cf56d7c33fd77b3d00a38235661275fd4b2452eea3863bc469a591ecfd24c770593c83960e35a057b8150e1ff1088dbd8ce770ec862c7af6c327b66178cf1a576c7e256f88edba8ef322e0baa9e9d3c19d8929a57d589f6c2773927ec660d795042f8aecff9853cc70b9db9bf230cf8d2bee8a6c5 -->

natgeo-wong commented:
[noblock] I am requesting permission to bypass the naming guidelines for this package because it deals with the downloading of the [ETOPO](https://www.ncei.noaa.gov/products/etopo-global-relief-model) dataset, which is named in all-caps.

JuliaRegistrator committed:
New package: ETOPO v0.1.0

UUID: eb8fb1ab-fde0-4497-ad36-194d38f2cde7
Repo: https://github.com/GeoRegionsEcosystem/ETOPO.jl.git
Tree: aebdccf0e33879a7a96a19aaa931bc297de5e9af

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
